yang/ietf-l3vpn-ntw.yang

```diff
@@ -2125,6 +2141,84 @@ module ietf-l3vpn-ntw {
                           "Indicates whether IPv4, IPv6, or both
                            address families are to be activated.";
                       }
+                      container timers {
+                        description
+                          "Indicates the RIP timers.";
+                        reference
+                         "RFC 8695: A YANG Data Model for the 
```

Comment:
I would refer to RFC2453/RFC2080, instead.

Reply:
please see 77704cdb1183d3c6f7cfdb13ba3fb6b1d87803b2